frontend/js/src/pins/PinRecordingModal.tsx

```diff
@@ -111,7 +118,44 @@ export default NiceModal.create(
       },
       [recordingToPin, blurbContent]
     );
-
+    const updatePinnedRecordingComment = React.useCallback(
+      async (event: React.MouseEvent<HTMLButtonElement>) => {
+        event.preventDefault();
+        try {
+          if (rowId && recordingToPin && currentUser?.auth_token) {
+            const response = await APIService.updatePinRecordingBlurbContent(
+              currentUser.auth_token,
+              rowId,
+              blurbContent
+            );
+            if (!response.status) {
+              toast.error(
+                <ToastMsg
+                  title="Comment update failed"
+                  message="Something went wrong"
```

Comment:
It would be nice to show the user the actual error if the server returns one, to help with issue reports.